### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by Prabs09**
### Updated Online Manual Link.

### CODE REVIEW

File: Docs/Readme.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Open Rails is a free train simulator supporting the world's largest range of dig
 ## Documentation
 
 * [Read the FAQ](http://www.openrails.org/learn/faq/?utm_campaign=documentation&utm_source=readme&utm_medium=referral)
-* [Read the manual online](https://open-rails.readthedocs.io/en/1.3.1/) or the PDF included with the program, found in the main menu under "Documents"
+* [Read the manual online](https://open-rails.readthedocs.io/en/1.5.1/) or the PDF included with the program, found in the main menu under "Documents"
```
Comment (Looky1173) [inline Docs/Readme.md:13]:
```suggestion
* [Read the manual online](https://open-rails.readthedocs.io/en/latest/) or the PDF included with the program, found in the main menu under "Documents"
```

Makes the link always point to the latest documentation, so it does not need to be updated with each new release of OR.

**Review by Looky1173 — APPROVED** [review]
Looks good to me!

**Comment by cjakeman** [comment]
Hi Looky,

> Looks good to me!

Thanks for approving this.

There's no reason why, after approving, you should not go ahead and merge immediately. Will save someone else picking it up later and doing that.

Best wishes,

Chris

**Comment by Looky1173** [comment]
Hi Chris,

> There's no reason why, after approving, you should not go ahead and merge immediately. Will save someone else picking it up later and doing that.

Thanks for letting me know. I reasoned that since this is my first time approving an OR pull request, I should let someone more experienced from the Team merge it. :-)

Best regards,
Looky

**Pull request merged**